components/event_dispatcher/introduction.rst

```diff
@@ -81,17 +81,10 @@ Naming Conventions
 The unique event name can be any string, but optionally follows a few simple
 naming conventions:
 
-* use only lowercase letters, numbers, dots (``.``) and underscores (``_``);
-
-* prefix names with a namespace followed by a dot (e.g. ``kernel.``);
-
-* end names with a verb that indicates what action is being taken (e.g.
-  ``request``).
-
-Here are some examples of good event names:
-
-* ``kernel.response``
-* ``form.pre_set_data``
+* Use only lowercase letters, numbers, dots (``.``) and underscores (``_``);
+* Prefix names with a namespace followed by a dot (e.g. ``store.``);
+* End names with a verb that indicates what action has been taken (e.g.
+  ``order_placed``).
```

Comment:
The string of the event name contain any character that can be added to a simple PHP string. So there is no mandatory thing in the list.